From 49d6cb50da7d780b41f50f955803d64e695b1daf Mon Sep 17 00:00:00 2001
From: Alexandre Torgue <alexandre.torgue@st.com>
Date: Thu, 26 Sep 2019 12:00:34 +0200
Subject: [PATCH 212/214] cpufreq: dt-platdev: Add stm32mp157 SOC in the
 blacklist

A dedicated STM32 cpufreq driver is needed to read OTP data and select
available OPPs for STM32 MPU SOCs. Those OPPs are then registered to OPP
framework. At the end this driver register a cpufreq-dt device.
This commit allows it.

Upstream-Status: Denied Downstream stm32-gh v5.4-stm32mp [a375510e303563d856ace1f8fa26eeacc1ab6ee1]
Change-Id: I66159cfb2e8ab4abb627f3640c80eddece44b263
Signed-off-by: Alexandre Torgue <alexandre.torgue@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/144844
Reviewed-by: CITOOLS <smet-aci-reviews@lists.codex.cro.st.com>
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@st.com>
---
 drivers/cpufreq/cpufreq-dt-platdev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/cpufreq/cpufreq-dt-platdev.c b/drivers/cpufreq/cpufreq-dt-platdev.c
index 1200842c3da4..32f309fd6a57 100644
--- a/drivers/cpufreq/cpufreq-dt-platdev.c
+++ b/drivers/cpufreq/cpufreq-dt-platdev.c
@@ -133,6 +133,7 @@ static const struct of_device_id blacklist[] __initconst = {
 
 	{ .compatible = "st,stih407", },
 	{ .compatible = "st,stih410", },
+	{ .compatible = "st,stm32mp157", },
 
 	{ .compatible = "sigma,tango4", },
 
-- 
2.30.2

